src/Features/VisualBasic/Completion/KeywordRecommenders/Statements/FinallyKeywordRecommender.vb

```diff
@@ -41,5 +41,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.KeywordRecommenders.Stat
 
             Return SpecializedCollections.EmptyEnumerable(Of RecommendedKeyword)()
         End Function
+
+        Private Function IsInCatchOfTry(targetToken As SyntaxToken, tryBlock As TryBlockSyntax) As Boolean
```

Comment:
If you're bored this can be come something like targetToken.GetAncestors(Of CatchBlockSyntax).TakeUntil(...).